### PR TITLE
Fix golint failures pkg/kubelet/util/cache

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:
Fix golint failures

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by xichengliudui** [comment]
@vishh  I think it is wrong to modify in this way. May I ask how can I modify?

File: pkg/kubelet/util/cache/object_cache_test.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func TestAddAndGet(t *testing.T) {
 		val: "bar",
 	}
 	objectCache := NewFakeObjectCache(func() (interface{}, error) {
-		return nil, fmt.Errorf("Unexpected Error: updater should never be called in this test!")
+		return nil, fmt.Errorf("Unexpected Error: updater should never be called in this test! ")
```
[the remark this answers is not in the document]
Comment (xichengliudui) [reply pkg/kubelet/util/cache/object_cache_test.go]: golint has not found any error in running, please tell me what is wrong and I will fix it!

[the remark this answers is not in the document]
Comment (BenTheElder) [reply pkg/kubelet/util/cache/object_cache_test.go]: why did we put a space here?

[the remark this answers is not in the document]
Comment (xichengliudui) [reply pkg/kubelet/util/cache/object_cache_test.go]: if no space are added, run golint detection: `pkg/kubelet/util/cache/object_cache_test.go:49:26: error strings should not be capitalized or end with punctuation or a newline`

[the remark this answers is not in the document]
Comment (xichengliudui) [reply pkg/kubelet/util/cache/object_cache_test.go]: you can also choose to delete `!` or do you have a better idea?

[the remark this answers is not in the document]
Comment (BenTheElder) [reply pkg/kubelet/util/cache/object_cache_test.go]: removing the "!" seems better than skirting the lint by adding a space to the end :^)

[the remark this answers is not in the document]
Comment (xichengliudui) [reply pkg/kubelet/util/cache/object_cache_test.go]: I agree with！

**Review by fejta — APPROVED** [review]
/lgtm
/approve

**Pull request merged**